app/src/main/java/com/example/dine_and_donate/SearchDialogFragment.java

```diff
@@ -0,0 +1,74 @@
+package com.example.dine_and_donate;
+
+import android.app.Activity;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.graphics.Color;
+import android.graphics.drawable.ColorDrawable;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.AutoCompleteTextView;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.DialogFragment;
+import androidx.fragment.app.Fragment;
+
+import com.example.dine_and_donate.Activities.HomeActivity;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class SearchDialogFragment extends DialogFragment {
+
+    public static final String ORGS = "orgNames";
+    public HomeActivity mHomeActivity;
```

Comment:
I think that putting the "m" in front of the name only applies to private variables